requirements.txt

```diff
@@ -14,3 +14,4 @@
 
 requests
 urllib3
+pyjwt[crypto]
```

Comment:
This PR does add this dependency. I think it's a fairly popular package (4.3k stars on GitHub).

Reply:
Thanks for being vigilant on dependencies. Agreed that this one is probably fine. We also don't have to be  as strict for benchalerts as we do for datalogistik since this will be only installed in a handful of places (mostly in CI)